src/keysym.h

```diff
@@ -72,9 +74,31 @@
 /** Maximum Unicode keysym, correspoding to the maximum Unicode code point */
 #define XKB_KEYSYM_UNICODE_MAX    0x0110ffff
 
+#ifdef ENABLE_PRIVATE_APIS
```

Comment:
am I missing something here? `src/keysym.h` is an internal header anyway so we don't need a special `#ifdef` to keep the APIs private, right?

I tried removing this `#ifdef` here and `nm` doesn't list any of the symbols so we're good here.

Reply:
416 bytes on my machine. Not much, but I guess it is still good to keep the size low, especially if it does not require effort. I would like to investigate the cache misses some day.